app.py

```diff
@@ -10,7 +10,9 @@
 sys.path.append('translate/routers')
 
 import js_main as js_main
```

Comment:
The alias for imports with the same name are redundant. Delete the alias on those cases.

Reply:
got it